### ISSUE TITLE
Importing intrinsics gives "unresolved import"

### ISSUE THREAD

**Issue by jrmuizel** [rust-analyzer/rust-analyzer#6077]
```rust
pub use std::arch::x86_64::{
    __m128, __m128i, _mm_add_ps, _mm_cvtps_epi32, _mm_load_ps, _mm_load_ss, _mm_max_ps, _mm_min_ps,
    _mm_mul_ps, _mm_set1_ps, _mm_setzero_ps, _mm_shuffle_ps, _mm_store_si128,
};

fn main() {
    println!("Hello, world!");
}
```

This worked for me yesterday.

**Comment by jrmuizel** [comment]
Of note: it's rust-analyzer that's complaining. rustc has no problem.

**Comment by bjorn3** [comment]
I think rust-analyzer doesn't yet support `#[path="../../stdarch/crates/core_arch/src/mod.rs"] mod core_arch;`.

**Comment by matklad** [comment]
Hm, I think we do support this:

https://github.com/rust-analyzer/rust-analyzer/blob/f7144dcd52c663cefc789445fe561771d935fbf0/crates/hir_def/src/nameres/tests/mod_resolution.rs#L303-L305

For the start, I think adding some printfs around here woudl help:

https://github.com/rust-analyzer/rust-analyzer/blob/f7144dcd52c663cefc789445fe561771d935fbf0/crates/hir_def/src/nameres/mod_resolution.rs#L46-L53

**Comment by novacrazy** [comment]
Is there any update on this?

**Comment by dylanede** [comment]
Seems to be a duplicate of #3898, because the arch module is defined in a directory that is not within the root of the crate.

**Comment by ClydeHobart** [comment]
Is there a work around for this that utilizes the `#[path="..."]` attribute?

**Comment by matklad** [comment]
fixed by #9963
